### PR TITLE
{bio}[gompi/2022a] LoRDEC v0.9, GATB-Core v1.4.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3158.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/b0575fe7efac078a12167b56df92fc5f for a full test report.

File: easybuild/easyconfigs/l/LoRDEC/LoRDEC-0.9_GraphVector.patch [hunk 1]
```diff
@@ -0,0 +1,73 @@
+diff -ru lordec-src_0.9.orig/lordec-correct.h lordec-src_0.9/lordec-correct.h
```
Comment (branfosj) [inline easybuild/easyconfigs/l/LoRDEC/LoRDEC-0.9_GraphVector.patch:1]: Add description and author information for this patch.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/LoRDEC/LoRDEC-0.9_GraphVector.patch (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/LoRDEC/LoRDEC-0.9_GraphVector.patch:3]: fixed in b9f6633

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/aa2339119a31b8e318307a17cc17c410 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/b42f1e0f1611b51ec5e3b530e61a697a for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**